src/cadenza.js

```diff
@@ -64,6 +64,13 @@ globalThis.cadenza = Object.assign(
  * _Note:_ The GeoJSON geometry type "GeometryCollection" is currently not supported.
  */
 /** @typedef {[number,number,number,number]} Extent - An array of numbers representing an extent: [minx, miny, maxx, maxy] */
+/**
+ * @typedef {[string]} DisabledUiFeatures - An array of strings representing Cadenza UI features to disable
+ *
+ * _Note:_ Supported features are:
+ * * 'workbook-design' - Disable the designer
+ * * 'workbook-view-management' - Disable workbook layout/design editing
+ * */
```

Comment:
To me it would be more sensible have UiFeature as the type and then disabledUiFeatures would just be an array of those.

```javascript
/**
 * @typedef {'workbook-design' | 'workbook-view-management'} UiFeature - The name of a Cadenza UI feature
 *
 * Supported features are:
 * - `workbook-design`: The designer
 * - `workbook-view-management` - Add/Edit/Remove workbook views
 */
```
```javascript
   * @param {UiFeature[]} [options.disabledUiFeatures] - Cadenza UI features to disable
```

(Btw, an array of strings would be string[]. [string] is an array with exactly one string value.)

Reply:
Good idea.

Oops, didn't quite pay attention there.